tracker-pt/lang/nl_NL.ts

```diff
@@ -232,6 +232,30 @@ Don&apos;t roll or change position.</source>
         <source>Green only</source>
         <translation type="unfinished"></translation>
     </message>
+    <message>
+        <source>Red filter</source>
```

Comment:
The difference between extracting a single channel and bias against that color isn't clear to the end-user.

Reply:
Consider "Red bias" as naming.

Reply:
Agreed, the distinction between "Red only" and "Red filter" or "Red bias" is going to be hard to describe. It isn't a bias against the color, BTW, it's a bias against the others. So the meaning of "green filter" here is to filter out all colors other than green. Which sounds kind of like "green only".

Maybe "red/green/blue/cyan/yellow/magenta chroma key"? that's the term of art, technically.

Reply:
I'll change it to chroma key. That's the right name for this.